buildconfig/CMake/FindCxxTest.cmake

```diff
@@ -168,6 +168,15 @@ macro(CXXTEST_ADD_TEST _cxxtest_testname)
     # The TESTHELPER_SRCS variable can be set outside the macro and used to pass in test helper classes
     add_executable(${_cxxtest_testname} EXCLUDE_FROM_ALL ${_cxxtest_cpp_files} ${_cxxtest_h_files} ${TESTHELPER_SRCS} )
 
+    set (_misc_bin ${CMAKE_SOURCE_DIR}/external/src/ThirdParty/bin)
```

Comment:
Could we us the `THIRD_PARTY_DIR` variable to avoid repeating those paths?

https://github.com/mantidproject/mantid/blob/7fde850aa59d756838726590627be49352ed476e/buildconfig/CMake/Bootstrap.cmake#L19